### PR TITLE
Make a 'pachyderm-tap/pachctl' formula represent the latest stable release

### PR DESCRIPTION
**Pull request by acohen4**

### CODE REVIEW

**Review by jrockway — APPROVED** [review]
This looks like it will work!

File: etc/build/update_homebrew.sh [hunk 1]
```diff
@@ -20,6 +21,11 @@ pushd homebrew-tap
     git checkout -b "$BRANCH" || git checkout "$BRANCH"
     VERSION=$version ./update-formula.sh
     git add "pachctl@$MAJOR_MINOR.rb"
+    if [[ $stable ]]; then
+        cp "pachctl@$MAJOR_MINOR.rb" pachctl.rb
```
Comment (molinamelendezj) [inline etc/build/update_homebrew.sh:25]: Can't remember, but cp will over write file with out error if file exist right.

**Review by molinamelendezj — APPROVED** [review]
Looks good one small comment. Please cherry pick this too.

File: etc/build/update_homebrew.sh (hunk 1 above)
Comment (acohen4) [reply etc/build/update_homebrew.sh:25]: nah it will create <dest> if it doesn't exist

**Pull request merged**